tools/e2e-commons/pages/page-actions.js

```diff
@@ -93,10 +93,13 @@ export default class PageActions {
 	/**
 	 * Waits for page to reach the 'networkidle' load state or timeout in given ms
 	 *
+	 * TODO: Deprecate and remove this, see https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/no-networkidle.md
```

Comment:
@jeherve FYI. Not sure if there's anyone better than you to ping about stuff like this now that team JARVIS is no more.

Reply:
Yeah, but he's in another division now so Jetpack TODOs probably aren't for him to do anymore. (: